### PR TITLE
Added "Are you sure you want to mark all notifications as read?"

### PR DESCRIPTION
**Pull request by bogdanteodoru**
For fix https://github.com/flarum/core/issues/679

### CODE REVIEW

File: locale/core.yml [hunk 1]
```diff
@@ -259,6 +259,7 @@ core:
       mark_all_as_read_tooltip: => core.ref.mark_all_as_read
       title: => core.ref.notifications
       tooltip: => core.ref.notifications
+      mark_all_as_read_confirm_text: "Are you sure you want to mark all notifications as read?"
```
Comment (franzliedke) [inline locale/core.yml]: To be consistent, this should probably be called `mark_all_as_read_confirmation`.

/cc @dcsjapan

Comment (maelsoucaze) [inline locale/core.yml]: ... and should be added before `mark_all_as_read_tooltip` :) .

Comment (dcsjapan) [inline locale/core.yml]: > To be consistent, this should probably be called mark_all_as_read_confirmation.

> ... and should be added before mark_all_as_read_tooltip. :)

:+1:

**Comment by bogdanteodoru** [comment]
I have updated the name and moved the location for this (due to initial task changes)

**Comment by maelsoucaze** [comment]
It looks great.

Note to @tobscure: the year range should be updated in the LICENSE file if this PR is merged.

**Comment by franzliedke** [comment]
@maelsoucaze Done.

**Pull request merged**